src/handlers/excludeDependencyHandler.ts

```diff
@@ -29,9 +33,9 @@ async function excludeDependency(pomPath: string, gid: string, aid: string, ance
 
     const projectNode: ElementNode = projectNodes[0];
     const dependenciesNode: ElementNode | undefined = projectNode.children && projectNode.children.find(node => node.tag === XmlTagName.Dependencies);
-    const dependencyNode: ElementNode | undefined = dependenciesNode?.children && dependenciesNode?.children.find(node => node.children && node.children[1].tag === XmlTagName.ArtifactId && node.children[1].text === ancestorAid);
+    const dependencyNode: ElementNode | undefined = dependenciesNode?.children && dependenciesNode?.children.find(node => node.children && node.children[1].tag === XmlTagName.ArtifactId && node.children[1].text === rootAid);
     if (dependencyNode === undefined) {
-        throw new UserError("Cannot find the ancestor dependency where to insert Exclusions.");
+        throw new UserError("Cannot find the root dependency where to insert Exclusions.");
```

Comment:
"root dependency" doesn't make sense to developers. I suggest to use something like following statement to provide more helpful information.

```
Failed to find the dependency where {gid:aid} is introduced.
```

And it's not a `UserError`, which represents errors caused by malformed settings/input from user.

Reply:
use `vscode.window.showErrorMessage` instead now.

Reply:
Can `throw new Error(msg)` work? I assume it would also call `vscode.window.showErrorMessage` eventually?